src/gConsent/request/issueAccessRequest.test.ts

```diff
@@ -460,6 +471,7 @@ describe("issueAccessRequest", () => {
             normalizedAccessRequest.credentialSubject.hasConsent
               .forPersonalData[0],
           mode: normalizedAccessRequest.credentialSubject.hasConsent.mode[0],
+          inherit: "true",
```

Comment:
Do we need a test case for inherit === false?

Reply:
Never too sure: d6ad1c5fc7857e295c3292f8423a78af21fc8230